src/base-container/index.jsx

```diff
@@ -37,7 +39,12 @@ const BaseContainer = ({
       isBlocking
       variant="default"
       title="authn-component"
-      className="bg-light-200 authn-component__modal"
+      className={classNames(
+        'bg-light-200 authn-component__modal',
+        {
+          'authn-component__modal-full-height': isFullScreen,
```

Comment:
Like:
```suggestion
          'authn-component__modal-full-height': size === 'fullscreen',
```
And by doing this, we don't need to pass extra prop.